site/_includes/macros/cards/blog-card.njk

```diff
@@ -3,8 +3,11 @@
 {% macro blogCard(post) %}
 {% set pd = post.data %}
 <div class="blog-card rounded-lg pad-300 md:pad-400 width-full">
-
-  {% set thumbnail = pd.hero or pd.thumbnail %}
```

Comment:
You could also just change the order, to prefer thumbnail over hero: 

{% set thumbnail = pd.thumbnail or pd.hero %}